config/optional/views.view.civiremote_funding_combined_application_process_list.yml

```diff
@@ -1036,7 +1038,59 @@ display:
             fail: 'not found'
           validate_options: {  }
           must_not_be: false
-      filters: {  }
+      filters:
```

Comment:
Is this related to the export?

Reply:
this status filter was configured already in the dev environment but i guess its not related to this feature. i can remove it but will add it again to another feature, where a status filter was requested in another view (based on the experience with the filter in this view)